homeassistant/components/device_tracker/icloud.py

```diff
@@ -430,7 +430,9 @@ def lost_iphone(self, devicename):
         self.api.authenticate()
 
         for device in self.api.devices:
-            if devicename is None or device == self.devices[devicename]:
+            if devicename is (
+                None or str(device) == str(self.devices[devicename])
```

Comment:
by adding parentheses around the check, it will never check if devicename is None because it will first evaluatie `None or X` and None is always false and thus returns X

Reply:
Thanks, how should I write this to work then?

Reply:
Put the parenthesis around the whole expression after `if`, to be able to break the line.

Reply:
Thanks, doing so I end up with the exact same line length of the 1-line solution... How can I break the expression to have the `devicename is None` on a line and the `or str(device)...` on another?

Reply:
Is this valid?

Reply:
```py
for device in self.api.devices:
    if (devicename is None or
            str(device) == str(self.devices[devicename])):
        device.play_sound()
```